src/sentence-length.ts

```diff
@@ -37,6 +37,12 @@ export type Options = {
      * @deprecated use skipPatterns
      */
     exclusionPatterns?: string[];
+    /**
+     * Determine how to count string length.
+     * By default or set to "code units", count string by UTF-16 code unit(= using `String.prototype.length`).
+     * If set to "codepoints", count string by codepoint.
+     */
+    countBy?: "code units" | "codepoints";
```

Comment:
```suggestion
    countBy?: "codeunits" | "codepoints";
```

I think it would be better to align without spaces.

Reply:
511ec1691ff5ad90301c4e5730c74b9b77f178e5